app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  # logs in the user
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by(id: user_id)
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by(id: user_id)
+      if user&.authenticated?(cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
+    end
+  end
```

Comment:
I really don't understand why I am not supposed to use sessions in this case. 
Sessions will be used temporarily when a user doesn't tick the remember me checkbox while logging in.
`current_user` will then pick either `session` or the permanent `cookies`.
Please may you elaborate more to the change